<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLogin</class>
 <widget class="QWidget" name="CLogin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>18</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>578</width>
     <height>378</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>60</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>301</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>301</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>账号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_nunber">
            <property name="toolTip">
             <string/>
            </property>
            <property name="toolTipDuration">
             <number>0</number>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="accessibleName">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>密码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_password">
            <property name="maxLength">
             <number>32768</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>验证码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_checkcode"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_checkcode">
            <property name="minimumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>93</width>
              <height>28</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
              <underline>false</underline>
              <strikeout>true</strikeout>
              <stylestrategy>PreferDefault</stylestrategy>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>60</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rb_keeppw">
            <property name="text">
             <string>记住密码</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_autologin">
            <property name="text">
             <string>自动登录</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pb_login">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>登录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_commit">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>41</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string>注册</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>widget_3</zorder>
      <zorder>widget_5</zorder>
      <zorder>widget_6</zorder>
      <zorder>widget_7</zorder>
      <zorder>widget_4</zorder>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>*注册成功后，三天之内未登录，服务器将自动收回账号</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="wdg_background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*border-image:url(:/picture/0.webp)*/</string>
   </property>
  </widget>
  <zorder>wdg_background</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
